docs/install/parameters.md

```diff
@@ -32,3 +32,4 @@ Alternatively, you can define the following environment variables:
 | `PLM_CLONE_NUM_PARALLEL_COLLECTIONS` | Number of collections cloned in parallel | `2` |
 | `PLM_CLONE_NUM_READ_WORKERS` | Number of read workers for cloning | `NumCPU / 4` |
 | `PLM_CLONE_NUM_INSERT_WORKERS` | Number of insert workers for cloning | `NumCPU * 4` |
+| `PLM_MONGODB_CLI_OPERATION_TIMEOUT` | Maximum time to wait before timing out MongoDB client operations such as insert, update, delete. If the timeout is reached, the operation will fail.  | `5m` |  
```

Comment:
Hey, we can go with this name, but we could also go with updated `PCSM_*` prefix and maybe also change the rest of env var names. Wdyt?